### ISSUE TITLE
No html file generated

### ISSUE THREAD

**Issue by AmjadHD** [treeform/fidget#172]
When running `nim js areas.nim`, I only get a js file, no html.

**Comment by levovix0** [comment]
It is a normal nim compiler behavior, aren't it?

**Comment by AmjadHD** [comment]
> First `cd` to each folder then run `js` compile command then open the `.html` file in a browser

I guess I mistakenly thought it would generate a `.html` for me.
how would a minimal html file that runs this look like ?

**Comment by levovix0** [comment]
something like
```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <title>Fidget example</title>
</head>
<body>
  <script src="areas.js"></script>
</body>
</html>
```

**Comment by treeform** [comment]
That is correct Nim does not generate HTML files. You got to make a minimal one yourself.

Closing this issue, feel free to reopen if you still have questions.

**Comment by jyapayne** [comment]
@treeform this is an issue with the README. It says:

> First cd to each folder then run js compile command then open the .html file in a browser.

So you'd think there would be an html file. Maybe it would be helpful to just add that bit of html above to the doc to make it a bit easier on newcomers?

**Comment by treeform** [comment]
You right its probably is my issue. If you follow the emscriten process it generates the html file, if you follow the simple nim js it does not. I need to figure out the best way to do this.
